tests/Backend/Snowflake/TableDefinitionOperationsTest.php

```diff
@@ -826,6 +826,140 @@ public function testCreateTableDefaults(): void
+            'columns' => [
+                [
+                    'name' => 'id',
+                    'basetype' => 'numeric',
+                ],
+                [
+                    'name' => 'numeric',
+                    'basetype' => 'numeric',
+                ],
+                [
+                    'name' => 'BOOLEAN',
+                    'basetype' => 'BOOLEAN',
+                ],
+                [
+                    'name' => 'DATE',
+                    'basetype' => 'DATE',
+                ],
+                [
+                    'name' => 'FLOAT',
+                    'basetype' => 'FLOAT',
+                ],
+                [
+                    'name' => 'INTEGER',
+                    'basetype' => 'INTEGER',
+                ],
+                [
+                    'name' => 'STRING',
+                    'basetype' => 'STRING',
+                ],
+                [
+                    'name' => 'TIMESTAMP',
+                    'basetype' => 'TIMESTAMP',
+                ],
+            ],
+        ];
+
+        $runId = $this->_client->generateRunId();
+        $this->_client->setRunId($runId);
+
+        $newTableId = $this->_client->createTableDefinition($bucketId, $data);
+        $tableDetail = $this->_client->getTable($newTableId);
+        $this->assertSame([
+            'primaryKeysNames' => ['id'],
+            'columns' => [
+                [
+                    'name' => 'BOOLEAN',
+                    'definition' => [
+                        'type' => 'BOOLEAN',
+                        'nullable' => true,
+                    ],
+                    'basetype' => 'BOOLEAN',
+                    'canBeFiltered' => true,
+                ],
+                [
+                    'name' => 'DATE',
+                    'definition' => [
+                        'type' => 'DATE',
+                        'nullable' => true,
+                    ],
+                    'basetype' => 'DATE',
+                    'canBeFiltered' => true,
+                ],
+                [
+                    'name' => 'FLOAT',
+                    'definition' => [
+                        'type' => 'FLOAT',
+                        'nullable' => true,
+                    ],
+                    'basetype' => 'FLOAT',
+                    'canBeFiltered' => true,
+                ],
+                [
+                    'name' => 'id',
+                    'definition' => [
+                        'type' => 'NUMBER',
+                        'nullable' => false,
+                        'length' => '38,9',
+                    ],
+                    'basetype' => 'NUMERIC',
+                    'canBeFiltered' => true,
+                ],
+                [
+                    'name' => 'INTEGER',
+                    'definition' => [
+                        'type' => 'NUMBER',
+                        'nullable' => true,
+                        'length' => '38,0',
```

Comment:
testy mi prechadzaju ale jak to ze tu je `38,0` a nie `38,9` ?

Reply:
To platí jen pro basetype numeric pokud zadáš typ přímo tak tam bude to co tam dá SNFLK